### PR TITLE
i18n - pulling from tx

### PR DESCRIPTION
**Pull request by stejskalleos**

### CODE REVIEW

File: locale/fr/foreman_bootdisk.po [hunk 1]
```diff
@@ -256,7 +256,7 @@ msgid "Unable to generate disk %{kind} template: %{error}"
 msgstr "Impossible de générer le modèle %{kind} de disque : %{error}"
 
 msgid "Unable to generate disk template, %{kind} template not found."
-msgstr "Impossible de générer le modèle de disque, le modèle %%{kind} est introuvable."
+msgstr "Impossible de générer le modèle de disque, le modèle%{kind} est introuvable."
```
Comment (ekohl) [inline locale/fr/foreman_bootdisk.po:259]: Looks like this was intended
```suggestion
msgstr "Impossible de générer le modèle de disque, le modèle %{kind} est introuvable."
```

File: locale/ja/foreman_bootdisk.po [hunk 2]
```diff
@@ -77,28 +78,28 @@ msgid "Download subnet generic image"
 msgstr "サブネットの汎用イメージのダウンロード"
 
 msgid "Ensure %{file} is readable (or update \"Grub2 directory\" setting)"
-msgstr "%{file} が読み取り可能であることを確認します (または「Grub2 ディレクトリー」設定を更新します)"
+msgstr "%%{file} が読み取り可能であることを確認します (または「Grub2 ディレクトリー」設定を更新します)"
```
Comment (ekohl) [inline locale/ja/foreman_bootdisk.po:81]: The `%%{TAG}` appears to be a repeating problem. I think I've also seen this in Transifex, so it isn't some pulling problem.

**Review by ekohl — COMMENTED** [review]
There is no mechanism to push translations back from source to Transifex. The best way is to change it in Transifex, either via the UI or upload an updated .po file. Then pull again.

**Comment by stejskalleos** [comment]
@ekohl updated the PR

File: app/assets/javascripts/foreman_bootdisk/locale/fr/foreman_bootdisk.js [hunk 3]
```diff
@@ -109,7 +109,7 @@
         "L'hôte n'est pas en mode build, c'est pourquoi le modèle ne peut être rendu"
       ],
       "Unable to generate disk template, %{kind} template not found.": [
-        "Impossible de générer le modèle de disque, le modèle %%{kind} est introuvable."
+        "Impossible de générer le modèle de disque, le modèle%{kind} est introuvable."
```
Comment (ekohl) [inline app/assets/javascripts/foreman_bootdisk/locale/fr/foreman_bootdisk.js:112]: This line is correct in the .po file. I wonder why it's no longer in sync.

**Review by ekohl — CHANGES_REQUESTED** [review]
As for Japanese, I think it's best to fix it up in Transifex. Let me know if you need help with that.

**Comment by stejskalleos** [comment]
@ekohl I don't have permissions to edit strings in transifex, could you please add me to the project?

**Comment by ekohl** [comment]
I can't quite figure that out right now (the permission screens are not intuitive), so I quickly fixed it up myself. I'd expect that if you pull again, it is correct.

**Comment by stejskalleos** [comment]
@ekohl updated

**Comment by ekohl** [comment]
I still see the same problems. In the past I've seen some issues where I needed to run the command multiple times, probably because it didn't register the timestamp changes properly.

**Comment by stejskalleos** [comment]
>  I needed to run the command multiple times

Which one exactly? The one in the plugin or the two in the foreman?

**Comment by ekohl** [comment]
Good question. I used https://github.com/ekohl/theforeman-plumbing/blob/master/bulk-locale-update and I just reran that whole script. Didn't dive deeper. I think my script is now outdated since it doesn't call the JS bits, so be warned.

**Comment by ekohl** [comment]
Apologies for not paying attention to this for a while. When I now pull myself I don't see the `%%TAG` issue anymore so I think if you pull again, it should be correct.

**Comment by stejskalleos** [comment]
let's continue in https://github.com/theforeman/foreman_bootdisk/pull/151

**Pull request closed without merging**